scss/_header.scss

```diff
@@ -183,13 +189,83 @@ button {
         list-style: none;
         display: flex;
         padding: 0;
-        margin: 0 0 0 -20px;
+        margin: 0 0 0 -17px;
         li{
-            padding: 25px 20px;
+            padding: 25px 17px;
             a{
             text-decoration: none;
             color: #fff;
             }
         }
     }
+}
+
+.header {
+    .container-fluid {
+        padding: 0;
+    }
+}
+.navbar {
+    padding: 0;
+}
+.navbar {
+    background-color: transparent !important;
+}
+.nav-link {
+    padding: 0;
+}
+.navbar-light .navbar-nav .nav-link {
+    color: #fff;
+}
+.navbar-light .navbar-nav .active > .nav-link {
+    color: #fff;
+}
+.navbar-light .navbar-nav .nav-link:hover {
+    color: $color-brand-primary;
+}
+.navbar-light .navbar-toggler {
+    background-color: $color-brand-primary;
+    border-color: $color-brand-secondary;
+}
+
+// container sizes:
```

Comment:
Please, notice. It's already exists in scss/_layout.scss
https://github.com/FoxicUA/Foxtar_team/blob/4575e86e7f1bfff67db15dec174ee97b6e4b1b4b/scss/_layout.scss#L32